packages/linux-drivers/amlogic/avl6862-aml/package.mk

```diff
@@ -0,0 +1,32 @@
+################################################################################
+#      This file is part of LibreELEC - https://libreelec.tv
```

Comment:
Is this package just for the amlogic-dvb.service? If so I'd suggest to put the service to filesystem/...

Reply:
This service for S905 and S912 devices. I think it's better to have single amlogic-dvb.service definition as double it in filesystem/.. for each device.

Reply:
OK you are right